platform/core/src/services/CustomizationService/CustomizationService.test.js

```diff
@@ -1,313 +1,399 @@
-import CustomizationService, { CustomizationType, MergeEnum } from './CustomizationService';
-import log from '../../log';
-
-jest.mock('../../log.js', () => ({
-  info: jest.fn(),
-  warn: jest.fn(),
-  error: jest.fn(),
-}));
+// File: CustomizationService.registrationAndOperations.test.js
+import CustomizationService, { CustomizationScope } from './CustomizationService';
```

Comment:
Can you add a test for:
* inheritance of values
* transformed customization
* register of set of values in default, where both sets gets applied - this can be used to apply different initial configuration sets